dace/codegen/targets/fpga.py

```diff
@@ -69,6 +69,7 @@ def is_fpga_kernel(sdfg, state):
             return False
     return True
 
+
 def is_hbm_array(array: dt.Data):
```

Comment:
This is still used in other places, right?

Reply:
Yes there are like 2 or 3 locations